service/src/main/java/bio/terra/workspace/app/configuration/external/FeatureConfiguration.java

```diff
@@ -30,4 +31,11 @@ public void setAzureEnabled(boolean azureEnabled) {
   public void logFeatures() {
     logger.info("Feature: azure-enabled: {}", isAzureEnabled());
   }
+
+  /** Common test and throw for Azure enabled */
+  public void azureEnabledCheck() {
+    if (!isAzureEnabled()) {
+      throw new AzureNotImplementedException("Azure features are not enabled");
```

Comment:
shall we make the error message sth like "If you want to enable it, set ??? flag to be true?"

Reply:
I was imagining this error showing up in alpha, staging, or prod if someone is trying to run in the wrong context.